Havit/Havit/Screens/CategoryContents/View/Cell/CategoryContents1xNCollectionViewCell.swift

```diff
@@ -76,62 +94,77 @@ final class CategoryContents1xNCollectionViewCell: BaseCollectionViewCell {
     }
     
     override func render() {
-        contentView.addSubViews([mainImageView, titleLabel, subtitleLabel, dateLabel, linkLabel, alarmLabel, moreButton, isReadImageView])
+        contentView.addSubViews([mainImageView, titleLabel, subtitleLabel, dateLabel, linkLabel, alarmLabel, moreButton, isReadImageView, borderView])
         mainImageView.addSubview(alarmImageView)
         
         mainImageView.snp.makeConstraints {
             $0.top.equalTo(contentView).offset(17)
             $0.leading.equalTo(contentView).offset(16)
-            $0.trailing.equalTo(contentView).offset(-16)
+            $0.trailing.equalTo(contentView).inset(16)
             $0.height.equalTo(184)
         }
         
         alarmImageView.snp.makeConstraints {
             $0.leading.equalTo(mainImageView)
             $0.top.equalTo(mainImageView)
-            $0.width.equalTo(28)
-            $0.height.equalTo(28)
+            $0.width.equalTo(40)
+            $0.height.equalTo(40)
```

Comment:
p1;

```suggestion
            $0.width.height.equalTo(40)
```
이 부분은 이렇게 쓸 수 있겠네요!

Reply:
3e0c780feff586bc41